guider/guider.py

```diff
@@ -10565,7 +10565,7 @@ def printHelp():
                 t:thread | C:wfc | s:stack | w:wss | d:disk
                 P:Perf | i:irq | S:pss | u:uss | f:float
                 a:affinity | r:report | W:wchan | h:handler
-                f:float | R:freport
+                f:float | R:freport | v:visualization(elastic stack)
```

Comment:
what do you think about changing it like this?
it may require to add "E" option into isEffectiveEnableOption() function. 

> v:visualization(elastic stack)

to 

> E:elasticsearch

Reply:
Since 'e' and 'E' are on one line, i think it is not clear.
So I used 'v' instead of 'E'.
But if you want to change option, I will change that to your opinion. 
I agree that.